### ISSUE TITLE
Print out a success message when no errors are found.

### ISSUE THREAD

**Issue by zck** [purcell/package-lint#50]
I've been linting some of my code, and I got to a point where there are no errors or warnings. Yay!

But there isn't a message saying that everything is fine, so the package leaves an empty buffer. This is indistinguishable from the linter not working at all. Perhaps we could add a message like "Everything looks great here!" or "You did it! Now get a cookie!"

If you'd be open to this, I'm happy to add the functionality myself.

**Comment by purcell** [comment]
Are you talking about when you run the linter automatically? If so, that would be a great idea. A good message might be "No issues found in this file."

I think we should make sure that there are no messages displayed when `package-lint` is run in batch mode, though, to be consistent with general linter behaviour.

**Comment by Fanael** [comment]
I assume you want something along the lines of this?

```diff
diff --git a/package-lint.el b/package-lint.el
index 1922711..dd7ab55 100644
--- a/package-lint.el
+++ b/package-lint.el
@@ -579,7 +579,11 @@ Current buffer is used if none is specified."
       (let ((buffer-read-only nil))
         (erase-buffer)
         (pcase-dolist (`(,line ,col ,type ,message) errs)
-          (insert (format "%d:%d: %s: %s\n" line col type message))))
+          (insert (format "%d:%d: %s: %s\n" line col type message)))
+        (when (bobp)
+          ;; If we're at the beginning, the buffer is still empty, so there were
+          ;; no issues to report.
+          (insert "No issues found.")))
       (special-mode)
       (view-mode 1))
     (display-buffer buf)))
```

**Comment by zck** [comment]
Yeah, that's pretty much what I was going to code up.

While I'm thinking about it, perhaps there could be a count of the number of errors and warnings, and present that? Might be nice to give context as to how far you've come, if, for example, you fix some stuff and go from 63 problems to 48.

**Comment by Fanael** [comment]
Sure, it's a good idea, implemented.
